rules/rules-reviewed/azure/tests/azure-aws-config.windup.test.xml

```diff
@@ -57,6 +57,36 @@
                     <fail message="aws-secrets-config hint was not found!" />
                 </perform>
             </rule>
+            <rule id="azure-aws-config-test-06000">
```

Comment:
@KaiqianYang , Just a small suggestion: could you please adjust the test case numbers to align with the rule numbers? This would help enhance clarity and maintain our structure.

Reply:
Sure, done. @kthatipally